### PR TITLE
Update sql database project commands

### PR DESCRIPTION
**Pull request by kisantia**
- Removing sql database project's open and create new project from the command palette
- Updating create project from database to add the newly created project to the workspace and focus on the projects view

### CODE REVIEW

File: extensions/data-workspace/src/main.ts [hunk 1]
```diff
@@ -14,28 +14,32 @@ export function activate(context: vscode.ExtensionContext): void {
...
-				canSelectFolders: false,
-				canSelectMany: false,
-				defaultUri: vscode.Uri.file(path.dirname(vscode.workspace.workspaceFile.path)),
-				openLabel: SelectProjectFileActionName,
-				filters: filters
-			});
-			if (!fileUris || fileUris.length === 0) {
-				return;
+			if (uri) {
+				await workspaceService.addProjectsToWorkspace([uri]);
```
Comment (udeeshagautam) [inline extensions/data-workspace/src/main.ts:22]: in open from DB flow (and other flows) we will have vscode.workspace.workspaceFile ? as in the condition will be true?

File: extensions/sql-database-projects/package.json [hunk 2]
```diff
@@ -165,10 +165,12 @@
     "menus": {
       "commandPalette": [
         {
-          "command": "sqlDatabaseProjects.new"
+          "command": "sqlDatabaseProjects.new",
```
Comment (alanrenmsft) [inline extensions/sql-database-projects/package.json:168]: these 2 commands are no longer needed, right?

File: extensions/sql-database-projects/src/controllers/projectController.ts [hunk 3]
```diff
@@ -657,7 +657,9 @@ export class ProjectsController {
 			let fileFolderList: string[] = model.extractTarget === mssql.ExtractTarget.file ? [model.filePath] : await this.generateList(model.filePath); // Create a list of all the files and directories to be added to project
 
 			await project.addToProject(fileFolderList); // Add generated file structure to the project
-			await this.openProject(vscode.Uri.file(newProjFilePath));
+
```
Comment (alanrenmsft) [inline extensions/sql-database-projects/src/controllers/projectController.ts:660]: I think we should rename this method and the command to : createProjectFromDatabase

File: extensions/data-workspace/src/main.ts (hunk 1 above)
Comment (alanrenmsft) [reply extensions/data-workspace/src/main.ts:22]: I think it is ok for Kim to assume a workspace already exists, Sakshi's work on the add project dialog should implement the workspace creation logic. and we can update this scenario after that.

File: extensions/sql-database-projects/package.json (hunk 2 above)
Comment (kisantia) [reply extensions/sql-database-projects/package.json:168]: will the data workspace create new project not call this eventually? I also just realized that SSDT projects aren't getting updated with the necessary .NET Core things when added in the new projects view(this used to happen in openProject() in the sql database extension). Created #12601 to track fixing this

Comment (alanrenmsft) [reply extensions/sql-database-projects/package.json:168]: add project dialog will call the project provider api, not through command.

File: extensions/sql-database-projects/src/controllers/projectController.ts (hunk 3 above)
Comment (kisantia) [reply extensions/sql-database-projects/src/controllers/projectController.ts:648]: renamed all the remaining "import database" to create project from database

File: extensions/sql-database-projects/package.json (hunk 2 above)
Comment (kisantia) [reply extensions/sql-database-projects/package.json]: ok I removed the commands for open, new, and close. Will clean up ProjectsController.openProject() and ProjectsController.newProject() later when the project provider api gets updated

File: extensions/sql-database-projects/package.json [hunk 4]
```diff
@@ -330,31 +303,31 @@
...
       "objectExplorer/item/context": [
         {
-          "command": "sqlDatabaseProjects.importDatabase",
+          "command": "sqlDatabaseProjects.createProjectFromDatabase",
           "when": "connectionProvider == MSSQL && nodeType && nodeType == Database && mssql:engineedition != 11",
           "group": "export"
         },
         {
-          "command": "sqlDatabaseProjects.importDatabase",
+          "command": "sqlDatabaseProjects.createProjectFromDatabase",
```
Comment (alanrenmsft) [inline extensions/sql-database-projects/package.json:311]: the command contribution can be simplified to just one item per host if you use regex like I did, nodeType equals database or server type

File: extensions/data-workspace/src/dataworkspace.d.ts [hunk 5]
```diff
@@ -13,7 +13,21 @@ declare module 'dataworkspace' {
...
+		/**
+		 * Add projects to the workspace
+		 * @param projectFiles Uris of project files to add
+		 */
+		addProjectsToWorkspace(projectFiles: vscode.Uri[]): Promise<void>
+
+		/**
+		 * Change focus to Projects view
+		 */
+		focus(): void;
```
Comment (alanrenmsft) [inline extensions/data-workspace/src/dataworkspace.d.ts:30]: I would call this showProjectsView()

**Review by alanrenmsft — APPROVED** [review]
Left a couple comments, otherwise looks good.

**Review by SakshiS-harma — APPROVED** [review]

**Pull request merged**